sw/device/tests/sim_dv/uart_tx_rx_test.c

```diff
@@ -539,11 +539,11 @@ bool test_main(void) {
       kTopEarlgreyPinmuxMioOutIob5, kTopEarlgreyPinmuxOutselUart1Tx);
   // TODO: the UARTs below still need to be mapped to the correct location.
```

Comment:
Do I understand correctly that after this round of patches, we should still align the pin mapping with the main usecase? For instance, I think I also saw that the `sysrst_ctrl` signals are currently not all mapped to the IOR* bank yet.

Reply:
Yes.